src/uk/gov/hmcts/contino/YarnBuilder.groovy

```diff
@@ -88,6 +101,7 @@ class YarnBuilder extends AbstractBuilder {
     finally {
       steps.archiveArtifacts allowEmptyArchive: true, artifacts: "functional-output/$browser*/*"
       steps.saucePublisher()
+      savePodLogs("crossbrowser-${browser}")
```

Comment:
i'm not sure if it's a safe assumption that you will be logged into the cluster context for this.

Reply:
Yes, probably not.

I should have made it clear that this is a Work In Progress. I don't expect any of this to work just yet.

I opened the PR just to simplify "on the fly" reviews when I engage with others.